test/onigumo_test.exs

```diff
@@ -70,6 +70,39 @@ defmodule OnigumoTest do
     assert(loaded_urls == @urls)
   end
 
+  test("get response by HTTP request") do
+    expect(HTTPoisonMock, :get!, &get!/1)
+
+    url = Enum.at(@urls, 0)
+    get_response =  Onigumo.get_url(url, HTTPoisonMock)
+    expected_response = %HTTPoison.Response{
+      status_code: 200,
+      body: body(url)
+    }
+    assert(get_response == expected_response)
+  end
+
+  test("extract body from URL response") do
+    url = Enum.at(@urls, 0)
+    response = %HTTPoison.Response{
+      status_code: 200,
+      body: body(url)
+    }
+    get_body = Onigumo.get_body(response)
+    expected_body = body(url)
+    assert(get_body == expected_body)
+  end
+
+  @tag :tmp_dir
+  test("write response to file", %{tmp_dir: tmp_dir}) do
+    output_path = Path.join(tmp_dir, @output_path)
+    response = "Response!"
```

Comment:
I’d swap these two lines so their order would match the argument order of `write_response/2` below.